webpack/assets/javascripts/react_app/components/Bookmarks/index.js

```diff
@@ -0,0 +1,24 @@
+import { connect } from 'react-redux';
```

Comment:
Same comment as before about the filename :)

Reply:
I think this is ok to keep as is - `index.js` is actually meant to export the connected component if the component needs to access the store. That way we have `MyComponent/MyComponent.js` without wiring to store (useful for tests) and we connect it to redux in `MyComponent/index.js` - [SearchBar](https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/SearchBar/index.js) or [Breadcrumbs](https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/BreadcrumbBar/index.js) have this structure.

Reply:
@xprazak2 Right, I think I might have been trying to say the same thing :)